disconnect-blacklist.json

```diff
@@ -1,9000 +1,8774 @@
 {
-    "license": "Copyright 2010-2015 Disconnect, Inc. / This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. / This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details. / You should have received a copy of the GNU General Public License along with this program. If not, see <http://www.gnu.org/licenses/>.",
-    "categories": {
-        "Disconnect": [
-            {
-                "Facebook": {
-                    "http://www.facebook.com/": [
-                        "facebook.com",
-                        "facebook.de",
-                        "facebook.fr",
-                        "facebook.net",
-                        "fb.com",
-                        "atlassolutions.com",
-                        "friendfeed.com"
-                    ]
-                }
-            },
-            {
-                "Google": {
-                    "http://www.google.com/": [
-                        "2mdn.net",
-                        "admeld.com",
-                        "admob.com",
-                        "cc-dt.com",
-                        "destinationurl.com",
-                        "developers.google.com",
-                        "doubleclick.net",
-                        "gmail.com",
-                        "google-analytics.com",
-                        "adwords.google.com",
-                        "mail.google.com",
-                        "inbox.google.com",
-                        "plus.google.com",
-                        "plusone.google.com",
-                        "voice.google.com",
-                        "wave.google.com",
-                        "googleadservices.com",
-                        "googlemail.com",
-                        "googlesyndication.com",
-                        "googletagservices.com",
-                        "invitemedia.com",
-                        "orkut.com",
-                        "postrank.com",
-                        "smtad.net",
-                        "teracent.com",
-                        "teracent.net",
-                        "ytsa.net"
-                    ]
-                }
-            },
-            {
-                "ItIsATracker": {
```

Comment:
The test entry has been removed from the list. Is there any way you can add it to the upstream list too?

Perhaps it should be called "Mozilla Test Entry" instead of "ItIsATracker".